lib/buildkite/test_collector/test_links_plugin/reporter.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Buildkite::TestCollector::TestLinksPlugin
+  class Reporter
+    RSpec::Core::Formatters.register self, :dump_failures
+
+    def initialize(output)
+      @output = output
+    end
+
+    def dump_failures(notification)
```

Comment:
@niceking I've updated this to `dump_failures` now! I still need to do the check on line 13 though as this method is called even when there are no failures 🤷 But it does cut down a lot of the extra stuff I was doing with assigning the failures to an array.